### ISSUE TITLE
Showing shortkey as the mouse point to an option laid out in Toolbar.

### ISSUE THREAD

**Issue by PranavBhattarai** [giuspen/cherrytree#2106]
_TLDR; Add a shortkey alongside the description of Toolbar options in the bracket similar to what VSCode and many other editors/browsers do._

CherryTree does a pretty good job showing all the shortkeys whenever the user clicks any options in the top menus, like this:
![image](https://user-images.githubusercontent.com/32763112/187083936-d2caedf9-e161-4892-bb00-b80f688604e2.png)
For users who don't have brilliant memory but still want to be a "keyboard-driven" user to get the job done faster, this kind of visible shortkey alongside its option is very helpful. We **_slowly_** memorize such shortkeys depending on how often we use some of the options.

Why not bring this feature to the next level? Similar to what VS code does.
![image](https://user-images.githubusercontent.com/32763112/187083735-8ef66bfd-c7f0-4bbe-bca7-535a3e381a86.png)
Showing keyboard shortkeys are not limited to **options inside menubar**, but also the icons outside the menu bar _if there is a shortkey_. But in CherryTree it's not the case, as you can see:
![image](https://user-images.githubusercontent.com/32763112/187084330-04d4209d-606b-4732-ac94-90ad98fb4428.png)
If I want to **intend** a paragraph, then I either have to memorize its shortkey after giving a visit to Preference->KeyboardShortcuts->Indent or I have to click intend every single time whenever I want.

Memorizing 1 or 4 of your favorite shortkey is not a problem, but it's _more_ about ** how easily users can read their shortkey** on demand. Because there are numerous shortkey, sometimes one is used quite a lot, and sometimes it's barely touched. It's neither feasible to memory ALL shortkey.

**Comment by giuspen** [comment]
Makes sense to add the shortcut to the tooltip, I'll give it a try

**Comment by giuspen** [comment]
ok will be in upcoming 0.99.49

**Comment by PranavBhattarai** [comment]
Thanks a lot.
